src/PendingObjects/TestCall.php

```diff
@@ -140,7 +152,8 @@ public function skip($conditionOrMessage = true, string $message = ''): TestCall
 
         $condition = is_callable($condition)
             ? $condition
-            : function () use ($condition) { /* @phpstan-ignore-line */
+            : function () use ($condition) {
+                /* @phpstan-ignore-line */
```

Comment:
Looks like this was accidentally reformatted.
```suggestion
            : function () use ($condition) { /* @phpstan-ignore-line */
```

Reply:
Fixed!